src/html.php

```diff
@@ -113,7 +113,9 @@ function print_card($card_id, $cardfront, $cardback, $backvisible = true)
         $card_id,
         $cardfront
     );
-    echo "<p class=\"cardbacktitle\">" . $l["Back"] . "</p>\n";
+    echo "<p class=\"cardbacktitle\">" . $l["Back"] . 
+        "<button class=\"speakButton\" onclick=\"javascript:event.preventDefault(); 
+        readCardBack(" . $card_id . ")\">Play back side</button></p>\n";
```

Comment:
Instead of "Play back side", make a new entry in `$l` and add translations in `html_en.php` and `html_sv.php`.